frontend/src/components/common/ErrorPage/ErrorPage.tsx

```diff
@@ -63,6 +72,34 @@ export default function ErrorComponent(props: ErrorComponentProps) {
           message
         )}
       </Grid>
+      {!!error?.stack && (
+        <Grid item xs={12}>
+          <Box
+            sx={{
+              width: '100%',
+              maxWidth: 800,
+            }}
+          >
+            <Accordion>
+              <AccordionSummary expandIcon={<Icon icon="mdi:chevron-down" />}>
+                <Typography>{t('translation|Error Details')}</Typography>
+              </AccordionSummary>
+              <AccordionDetails>
+                <Box textAlign="right">
+                  <Button
+                    onClick={() => {
+                      navigator.clipboard.writeText(error.stack!);
+                    }}
+                  >
+                    {t('translation|Copy')}
+                  </Button>
+                </Box>
+                <Typography variant="body2">{error.stack}</Typography>
```

Comment:
Using a `pre` tag would render text a bit nicer
`<Typography variant="body2" component="pre" align="left" overflow="auto">`

![image](https://github.com/user-attachments/assets/d8c2c154-4533-4b8e-824d-bde3b8784572)

Reply:
Good point. I added the pre and alignment, but made it wrap instead of the overflow.